src/main/java/com/idrsolutions/microservice/Individual.java

```diff
@@ -70,7 +72,8 @@ public void doError(final int errorCode) {
     String toJsonString() {
         final StringBuilder json = new StringBuilder();
         json.append("{\"state\":\"").append(state).append("\"")
-                .append(errorCode != null ? ",\"errorCode\":" + errorCode : "");
+                .append(errorCode != null ? ",\"errorCode\":" + errorCode : "")
+                .append(errorMessage != null ? ",\"error\":" + errorMessage : "");
```

Comment:
String in the JSON should be enclosed by double quotes
```suggestion
                .append(errorMessage != null ? ",\"error\":\"" + errorMessage + "\"" : "");
```